docs/howto/stack_version_support.md

```diff
@@ -0,0 +1,80 @@
+Intro...
+
+TBD: When to reconsider the version constraint.
+
+
+Support for all versions since the release of the package
+----
+Example: `>=7.16.0`.
+
+With this approach, the released package can be used in any version since the
+release of the package.
+
+Pros:
+* No need to keep multiple development branches, no need for backports.
+
+Cons:
+* False premise, difficult to ensure compatibility with future majors.
+
+Recommendation: Not recommended.
```

Comment:
Not recommended. We replaced this constraint some time ago.

Reply:
++

Reply:
Adding this.